src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -2885,6 +2885,7 @@ export interface Bladeburner {
    *
    * Attempts to start the specified Bladeburner action.
    * Returns true if the action was started successfully, and false otherwise.
+   * Example: ns.bladeburner.startAction("Contracts", "Tracking")
```

Comment:
Examples should use the `@example` jsdoc syntax

Reply:
Is this correct now?

Reply:
Looks good, thanks. I will try to get you feedback on the other PRs tonight